docs/supported-mutators.md

```diff
@@ -320,3 +321,56 @@ Strings and literals identified to a regex are mutated in the following way:
 | `--a`    | `++a`   |
 
 [🔝 Back to Top](#)
+
+## Math Methods
+|      Original           |         Mutated         |
+| ----------------------- | ----------------------- |
+| `Acos()`                  | `Acosh()`                 |
```

Comment:
These all look like C# math functions, I don't know if for example Java has the same methods. Perhaps it's cleanest to do a similar thing to method expressions and make a C# category for math methods?

Reply:
We can add the categories as soon as a second language support this right? Because for now C# is the only category

Reply:
Yes, but this now reads as though this should be the implementation for any language that wants to add math methods, while that's not the case. There should at least be some indication that this is C#-specific

Reply:
Done